client/sites/tools/logs/components/details-modal.tsx

```diff
@@ -0,0 +1,39 @@
+import { Modal } from '@wordpress/components';
+import { useTranslate } from 'i18n-calypso';
+import { LogType, PHPLog, ServerLog } from 'calypso/data/hosting/use-site-logs-query';
+import DetailsModalPHP from './details-modal-php';
+import DetailsModalServer from './details-modal-server';
+
+interface Props {
+	item: PHPLog | ServerLog;
+	logType: LogType;
+	onClose: () => void;
+}
+
+export default function DetailsModal( { item, logType, onClose }: Props ) {
+	const translate = useTranslate();
+
+	if ( logType === LogType.PHP ) {
+		return (
+			<Modal
+				title={ translate( 'Log details' ) } /* Same title as dataviews action modal */
+				onRequestClose={ onClose }
+				focusOnMount="firstContentElement"
+				size="large"
+			>
```

Comment:
The `Modal` part is repeated, so we have 2 identical copies of this modal. We could have the conditional logic responsible only for rendering the inner modal contents. Unless there's a good reason for the duplication?

Reply:
I just thought it was more readable that way, which I prioritize over writing less characters :)

Reply:
I disagree, but leaving this to your judgment